README.md

```diff
@@ -5,9 +5,8 @@ packages, so this meta-repo exists to help you find what you're looking for.
 
 ## Resources
 
-- [Guides](https://glimmerjs.com/guides)
-- [API Docs](https://glimmerjs.com/api/)
-- [Glimmer Playground](https://glimmer-playground.netlify.com)
+- [Guides](https://github.com/glimmerjs/glimmer-experimental/blob/master/README.md)
+- [Glimmer Playground](https://glimmerjs.github.io/glimmer-experimental/)
```

Comment:
This is a slightly different thing (glimmerx's API's are different from glimmer.js's)

Reply:
@rwjblue, for sure, but currently the link to `https://glimmerjs.com/api/` is just broken.  Also at some point we updated the refs on `glimmerjs.com` to point to glimmerx things. These changes were just meant to align with those changes.

Reply:
If this doesn't seem like like a logical stand-in until we get more docs in place, I can close this PR.